src/kOS/Suffixed/Direction.cs

```diff
@@ -160,6 +160,16 @@ private void DirectionInitializeSuffixes()
             return new Vector(a.Rotation * (Vector3d)b);
         }
 
+        public static Vector operator +(Direction a, Vector b)
+        {
+            return new Vector(a.Rotation * (Vector3d)b);
+        }
+
+        public static Vector operator +(Vector b, Direction a)
+        {
+            return new Vector(a.Rotation * (Vector3d)b);
+        }
+
```

Comment:
Really, this used to work?   Sorry I thought you were adding it because it's new (green) code, not noticing that you're just mimicing previous functionality that worked by a different mechanism.

Sigh.  Then we have to support it even though it's quite wrong.